pkgs/top-level/all-packages.nix

```diff
@@ -10123,14 +10123,16 @@ in
 
   octave = callPackage ../development/interpreters/octave {
     python = python3;
+    mkDerivation = stdenv.mkDerivation;
     jdk = jdk8; # TODO: remove override https://github.com/NixOS/nixpkgs/pull/89731
   };
   octave-jit = callPackage ../development/interpreters/octave {
     python = python3;
     enableJIT = true;
+    mkDerivation = stdenv.mkDerivation;
     jdk = jdk8; # TODO: remove override https://github.com/NixOS/nixpkgs/pull/89731
   };
-  octaveFull = (lowPrio (libsForQt512.callPackage ../development/interpreters/octave {
+  octaveFull = (lowPrio (libsForQt5.callPackage ../development/interpreters/octave {
```

Comment:
I'd remove the `lowPrio` along with this change. It's a leftover from quiet some time ago.

Reply:
Done (not really sure if there's anything to be tested, let me know if so).